docs/references/ayx-plugin-cli.md

```diff
@@ -215,14 +215,25 @@ The plugin version.
 -   Type: String
 -   Permissible Values: Any
 
+#### \--omit-ui
+
+Do not generate UI artifacts for this plugin. UI artifacts can be generated later by using `generate-ui` command.
```

Comment:
TW review:

Don't generate UI artifacts for this plugin. You can generate UI artifacts later with the `generate-ui` command.